nhc/nhc-processor.go

```diff
@@ -1,16 +1,34 @@
 package nhc
 
 import (
+	"fmt"
+
 	"github.com/USACE/go-consequences/hazards"
+	"github.com/dewberry/gdal"
 	//"github.com/dewberry/gdal"
 )
 
 type nhcInundationData struct {
 	FilePath string
 }
-type LocationArgument struct {
-	X float64
-	Y float64
+
+type Location struct {
+	X    float64
+	Y    float64
+	SRID string
```

Comment:
I had thought about this, but decided against it, i do think this is the right thing to do.